### PR TITLE
Site: Add instructions for managing Calcite repos through GitHub

### PR DESCRIPTION
**Pull request by zabetak**

### CODE REVIEW

File: site/_docs/howto.md [hunk 1]
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
 The following sections are of interest to Calcite committers and in
 particular release managers.
 
+## Managing Calcite repositories through GitHub (for Calcite committers)
+
+Committers have write access to Calcite's 
+[ASF git repositories](https://gitbox.apache.org/repos/asf#calcite) hosting
+the source code of the project as well as the website.
+
+All repositories present on gitbox are available on GitHub with write-access
```
Comment (michaelmior) [inline site/_docs/howto.md:483]: gitbox -> Gitbox

File: site/_docs/howto.md [hunk 2]
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
...
+
+Committers have write access to Calcite's 
+[ASF git repositories](https://gitbox.apache.org/repos/asf#calcite) hosting
+the source code of the project as well as the website.
+
+All repositories present on gitbox are available on GitHub with write-access 
+enabled, including rights to open/close/merge pull requests and address issues.
+
+In order to exploit the GitHub services, committers should link their accounts
+(ASF, GitHub) via the [account linking page](https://gitbox.apache.org/setup/).
```
Comment (michaelmior) [inline site/_docs/howto.md:487]: accounts (ASF, GitHub) -> ASF and GitHub accounts

File: site/_docs/howto.md [hunk 3]
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
...
+
+Here are the steps:
+
+ * Set your GitHub ID into your [Apache ID profile](https://id.apache.org/).
```
Comment (michaelmior) [inline site/_docs/howto.md:491]: Set your GitHub ID into your [Apache ID profile](https://id.apache.org/)
vv
Set your GitHub username in your [Apache profile](https://id.apache.org/)

File: site/_docs/howto.md [hunk 4]
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
...
+ * Enable [GitHub 2FA](https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/)
+on your GitHub account. 
+ * Activating GitHub 2FA changes the authentication process and may affect the way you 
+ [access GitHub](https://help.github.com/en/github/authenticating-to-github/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line).
+You may need to establish personal access tokens or upload your public SSH key to GitHub depending on the 
+protocol (HTTPS vs. SSH) that you are using.
```
Comment (michaelmior) [inline site/_docs/howto.md:497]: (HTTPS vs. SSH) that you are using. ->  that you are using (HTTPS vs. SSH).

File: site/_docs/howto.md [hunk 5]
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
...
+ * Set your GitHub ID into your [Apache ID profile](https://id.apache.org/).
+ * Enable [GitHub 2FA](https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/)
+on your GitHub account. 
+ * Activating GitHub 2FA changes the authentication process and may affect the way you 
+ [access GitHub](https://help.github.com/en/github/authenticating-to-github/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line).
+You may need to establish personal access tokens or upload your public SSH key to GitHub depending on the 
+protocol (HTTPS vs. SSH) that you are using.     
+ * Merge your Apache and GitHub accounts using the [account linking page](https://gitbox.apache.org/setup/)
+(you should see 3 green checks in GitBox).
+ * Wait at least 30  minutes for an email inviting you to Apache GitHub Organization.
```
Comment (michaelmior) [inline site/_docs/howto.md:500]: "30  minutes" has an extra space

File: site/_docs/howto.md [hunk 6]
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
...
+ * Enable [GitHub 2FA](https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/)
+on your GitHub account. 
+ * Activating GitHub 2FA changes the authentication process and may affect the way you 
+ [access GitHub](https://help.github.com/en/github/authenticating-to-github/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line).
+You may need to establish personal access tokens or upload your public SSH key to GitHub depending on the 
+protocol (HTTPS vs. SSH) that you are using.     
+ * Merge your Apache and GitHub accounts using the [account linking page](https://gitbox.apache.org/setup/)
+(you should see 3 green checks in GitBox).
+ * Wait at least 30  minutes for an email inviting you to Apache GitHub Organization. 
+ * Accept invitation and verify that you are a [member of the team](https://github.com/orgs/apache/teams/calcite-committers/members).
```
Comment (michaelmior) [inline site/_docs/howto.md:501]: Accept invitation -> Accept the invitation

**Comment by michaelmior** [comment]
Forgive me for being rather pedantic. This looks great. Just a few small suggested changes.

**Comment by zabetak** [comment]
> Forgive me for being rather pedantic. This looks great. Just a few small suggested changes.

Thanks for the second pair of eyes @michaelmior and @chunweilei , much appreciated. I addressed the comments, if nothing comes up in the following days I will push it through ;)

**Pull request closed without merging**